src/main/java/com/ge/predix/metering/customer/ZoneBasedCustomerResolver.java

```diff
@@ -28,59 +28,59 @@
 
 public class ZoneBasedCustomerResolver implements CustomerResolver {
 
-    @Value("${METER_BASE_DOMAIN:localhost}")
-    private String serviceBaseDomain;
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(ZoneBasedCustomerResolver.class);
-
-    @Override
-    public Customer resolveCustomer(final HttpServletRequest request) {
-
-        String zone;
-        if (StringUtils.isNotEmpty(request.getHeader("Predix-Zone-Id"))) {
-            zone = request.getHeader("Predix-Zone-Id");
-
-        } else if (StringUtils.isNotEmpty(request.getHeader("X-Identity-Zone-Id"))) {
-            zone = request.getHeader("X-Identity-Zone-Id");
-        } else {
-
-            // get zone from subdomain
-            zone = getZoneNameFromRequestHostName(request.getServerName(), this.serviceBaseDomain);
-            if (StringUtils.isEmpty(zone)) {
-                LOGGER.debug("Failed to resolve customer from request because the request does not contain a"
-                        + " 'Predix-Zone-Id' or 'X-Identity-Zone-Id' header or a subdomain.");
-                return null;
-            }
-        }
-
-        return new Customer(null, zone);
-    }
-
-    /**
-     * @return empty string if requestHostname and baseDomain are identical, null if domain is not a sub-string of
-     *         requestHostname
-     */
-    static String getZoneNameFromRequestHostName(final String requestHostname, final String baseDomain) {
-
-        if (requestHostname.equals(baseDomain)) {
-            return "";
-        }
-
-        String regexPattern = "^(.*?)\\." + Pattern.quote(baseDomain) + "$";
-        Pattern pattern = Pattern.compile(regexPattern);
-        Matcher matcher = pattern.matcher(requestHostname);
-        if (!matcher.matches()) {
-            // There is no zone scope for this request. Return null
-            return null;
-        }
-
-        String subdomain = matcher.group(1);
-
-        return subdomain;
-    }
-
-    public void setServiceBaseDomain(final String serviceBaseDomain) {
-        this.serviceBaseDomain = serviceBaseDomain;
-    }
+	@Value("${METER_BASE_DOMAIN:localhost}")
```

Comment:
I don't see any changes in this class, revert back the formatting changes to avoid confusion.